docs/pages/includes/helm-reference/zz_generated.teleport-kube-agent.mdx

```diff
@@ -1499,7 +1499,7 @@ for more information.
 | `object` | `{}` |
 
 `extraLabels.pod` are labels to set on the Pods created by the
-Deployment or StatefulSet.
+Deployment, StatefulSet, or Job.
```

Comment:
Can you undo this change? Master has been broken today and [the fix is on the way](https://github.com/gravitational/teleport/pull/46243).

Reply:
sure, was typing a message to you already :)